pkg/kubelet/kubelet_test.go

```diff
@@ -50,7 +50,7 @@ func init() {
 	util.ReallyCrash = true
 }
 
-func newTestKubelet(t *testing.T) (*Kubelet, *dockertools.FakeDockerClient, *sync.WaitGroup) {
+func newTestKubelet(t *testing.T) (*Kubelet, *dockertools.FakeDockerClient, *sync.WaitGroup, *cadvisorMock) {
```

Comment:
This is probably OK for now, I added a TODO in case it grows more or changes in use.